packages/main/cypress/specs/SplitButton.cy.tsx

```diff
@@ -0,0 +1,137 @@
+import SplitButton from "../../src/SplitButton.js";
+import Menu from "../../src/Menu.js";
+import MenuItem from "../../src/MenuItem.js";
+
+describe("Split Button general interaction", () => {
+	it("tests inner buttons design", () => {
+		cy.mount(<SplitButton id="sbDefault" design="Default">Default</SplitButton>);
+
+		cy.get("#sbDefault")
+			.as("splitButton");
+
+		cy.get("@splitButton")
+			.shadow()
+			.find(".ui5-split-text-button")
+			.should("have.attr", "design", "Default");
+
+		cy.get("@splitButton")
+			.shadow()
+			.find(".ui5-split-arrow-button")
+			.should("have.attr", "design", "Default");
+	});
+
+	it("tests text button 'click' event", () => {
+		cy.mount(<SplitButton id="sbDefault" design="Default" onClick={cy.stub().as("clicked")}>Default</SplitButton>);
```

Comment:
id and design are obsolete. probably add stub for arrowClick to check that it's not fired?